File: tier1/csharp/6452ffae/Report.cs
using System.Collections.Generic;

public static class Report
{
    public static string Build(string code)
    {
        var holder = new List<string>();
        var same = holder;
        holder.Add(code);
        return Run("SELECT status FROM orders WHERE code = '" + same[0] + "'");
    }

    private static string Run(string statement) => statement;
}

File: tier1/java/63b24836/Report.java
import java.util.ArrayList;
import java.util.List;

public final class Report {
    public static String build(String code) {
        List<String> holder = new ArrayList<>();
        List<String> same = holder;
        holder.add(code);
        return run("SELECT status FROM orders WHERE code = '"
                   + same.get(0) + "'");
    }

    private static String run(String statement) {
        return statement;
    }
}
